### PR TITLE
Add book cover image

### PR DESCRIPTION
**Pull request by sebasdeldi**
image

Adds and styles book cover image

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -3,36 +3,47 @@
 .card {
   margin: 25px;
   padding: 25px;
+  display: flex;
 
-  .header {
-    margin-bottom: 50px;
+  .card-body{
+    order: 2;
```
Comment (jjuli93) [inline scss/detail.scss:9]: you could just define the img before the card-body on the html

Comment (jjuli93) [inline scss/detail.scss:9]: you could just define the img before the card-body on the html

Comment (sebasdeldi) [reply scss/detail.scss:9]: That's true, but I read that it's better to define the text context first due to SEO optmization reasons

Comment (sebasdeldi) [reply scss/detail.scss:9]: same answer as above

**Pull request merged**